scripts/main.js

```diff
@@ -100,6 +110,10 @@ function renderAppDetail (id) {
 
   .then(function (app) {
     $('#name-app').html(app.name)
+    // $('#name-app').data('id', ""+app.id)
```

Comment:
@gr2m @titay2 Thank you for the comments and feedback :D I'll clean up as soon as possible after solving one question. I was able to get 'app.id' by attr('data-id', ""+app.id), but with '.data('id', ""+app.id)'  and .prop('data-id', ""+app.id)', I was not able to get data-id. I thought three codes would work same, but it didn't. I wonder what are the mistakes, when I used '.data()' and '.prop()'. :)